02/app.js

```diff
@@ -1 +1,12 @@
-console.log('DOM');
\ No newline at end of file
+console.log('DOM');
+
+const linkList = document.querySelectorAll( '[data-url]' );
+
+if(linkList) {
```

Comment:
Podobnie tutaj - ilość elementów.

Reply:
W przypadku zadania wyżej ten zapis o którym mówisz ma słuszność, ale tutaj nie widzę powodu, gdy tablica będzie miała length=0 to forEach się po prostu nie wykona.